integration-cli/docker_api_test.go

```diff
@@ -71,3 +76,62 @@ func (s *DockerSuite) TestApiClientVersionOldNotSupported(c *check.C) {
 	c.Assert(status, check.Equals, http.StatusBadRequest)
 	c.Assert(len(string(body)), check.Not(check.Equals), 0) // Expected not empty body
 }
+
+func (s *DockerSuite) TestApiDockerApiVersionEnvVar(c *check.C) {
+	var svrVersion string
+
+	server := httptest.NewServer(http.HandlerFunc(
+		func(w http.ResponseWriter, r *http.Request) {
+			url := r.URL.Path
+			svrVersion = url
+		}))
+	defer server.Close()
+
+	// Test using the env var first
+	cmd := exec.Command(dockerBinary, "-H="+server.URL[7:], "version")
+	cmd.Env = append([]string{"DOCKER_API_VERSION=xxx"}, os.Environ()...)
+	out, _, _ := runCommandWithOutput(cmd)
+
+	c.Assert(svrVersion, check.Equals, "/vxxx/version")
+
+	if !strings.Contains(out, "API version:  xxx") {
+		c.Fatalf("Out didn't have 'xxx' for the API version, had:\n%s", out)
+	}
+
+	// Now use config file
```

Comment:
rest of the test should go into a separate method (or rename method name coz it says ...EnvVar)